docs/v8-perf-faq.md

```diff
@@ -0,0 +1,207 @@
+a large effect on the field CLS for long-lived pages.
+
+Lighthouse 8 introduces another adjustment to our CLS definition: including
+layout shift contributions from subframes. This brings our implementation in
+line with how CrUX computes field CLS. This comes with the implication that
+iframes (including ones you may not control) may be adding layout shifts which
+ultimately affect your CLS score. Keep in mind that the subframe contributions
+are [weighted by the in-viewport
+portion](https://github.com/WICG/layout-instability#cumulative-scores) of the
+iframe.
+
+### Why don't the numbers for TBT and FID match, if TBT is a proxy metric for FID?
+
+The commonality between TBT (collected in a lab environment) and FID (collected
+in a field context) is that they measure the impact on input responsiveness from
+long tasks on the main thread. Beyond that, they're quite different. FID
+captures the delay in handling the first input event of the page, whenever that
+input happened.  TBT roughly captures how dangerous the length of all the main
+thread's tasks are.
+
+It's very possible to have a page that does well on FID, but bad on TBT.  And
+it's slightly harder, but possible, to do well on TBT but bad on FID.   So, you
+shouldn't expect your TBT and FID measurements to correlate strongly. A
+large-scale analysis found their [Spearman's
+ρ](https://en.wikipedia.org/wiki/Spearman%27s_rank_correlation_coefficient) at
+about 0.40, which indicates a connection, but not one as strong as many would
+prefer.
+
+From the Lighthouse project's perspective, the current passing threshold for FID
+is quite lenient but more importantly, the percentile-of-record for FID (75th
+percentile) is not indicative of issues and the 95th percentile is a much
+stronger indicator of problematic interactions for this metric. We encourage
+user-centric teams to focus on the 95th percentile of all input delays (not just
+the first) in their field data in order to identify and address problems that
+surface just 10% of the time.
+
+Aside: The [Chrome 91 FID change for
+double-tap-to-zoom](https://chromium.googlesource.com/chromium/src.git/+/refs/heads/main/docs/speed/metrics_changelog/2021_05_fid.md)
+may be observable in your field metrics, with higher percentiles improving
+slightly.
+
+### Overall, what motivated the changes to the performance score?
+
+As it is anytime the Lighthouse score is updated, changes are made to reflect
+the latest in how to measure user experience quality holistically, accurately,
+and to focus attention on key priorities.
+
+Heavy JS and long tasks are a problem for the web that's
+[worsening](https://httparchive.org/reports/state-of-javascript#bytesJs). Field
+FID is currently too lenient and not sufficiently incentivizing action to
+address the problem. Lighthouse has historically weighed its interactivity
+metrics at 40-55% of the performance score and as interactivity is key to user
```

Comment:
```suggestion
metrics at 40-55% of the performance score and, as interactivity is key to user
```

(or go em dash: "of the performance score and—as interactivity is key to user experience—we maintain a...")